components/atom/checkbox/src/index.js

```diff
@@ -22,6 +22,7 @@ const AtomCheckbox = ({
   name,
   disabled,
   checked = false,
+  shouldRenderIcon = false,
```

Comment:
I may lack context but because the design of props of a checkbox component must expose a prop of if the icon is rendered or not. The developer who uses this component if he wants to render an icon passes it via prop. I don't see the need to add a boolean anymore

Reply:
I need this prop because AtomCheckbox is styled depending on `styledIcon`, and if I didn't return anything on this it renders the default checkbox style. I use `shouldRenderIcon` for this.